build.rs

```diff
@@ -44,6 +45,8 @@ const CHECK_CFG_EXTRA: &[(&str, &[&str])] = &[
     ),
 ];
 
+const MUSL_TIME64_ARCHS: &[&str] = &["arm", "mips", "powerpc", "x86"];
```

Comment:
Add a comment

`` /// Musl architectures that set `#define _REDIR_TIME64 1`. ``

I'd also call it `MUSL_REDIR_TIME64_ARCHES` to keep the lingo